### PR TITLE
Project of the Week: Kap

### PR DESCRIPTION
**Pull request by zeke**
This is the first PotW post! Planning to ship it Tuesday morning, around 10am PST.

cc @skllcrn @CodeTheory @matheuss @sindresorhus @malept

 [Kap Screenshot](https://getkap.co/)

### CODE REVIEW

**Comment by malept** [comment]
It just occurred to me: shouldn't an article about a screencast app include a short screencast GIF (made with Kap, obviously) that shows using the app? :smile:

File: _posts/2017-01-31-kap.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+## What is Kap and why did you create it?
+
+[Kap is an open-source screen recorder](https://getkap.co) built primarily for designers and developers to easily capture their work. People use it to share animated prototypes, document bugs, create silly GIFs and everything in-between.  
+
+We've seen people of all ages and backgrounds use it in educational settings, screencasts, tutorials... the list goes on. Even to create production assets! We're completely blown away by how well received our little side project has been.
+
+Why did we build it? That's a very good question, it's not like there's a lack of screen recorders out there! We felt the alternatives were either too complex, too expensive or too limited. Nothing felt *just right* for our everyday needs. We also think it's great when the tools we use to do our work are open-source, that way everyone can help shape them. [Building Kap ended up being just as much about what we didn't do](https://medium.com/wulkano-friends/from-idea-to-product-and-beyond-a12850403c38). It's all in the details, an accumulation of small improvements that became the outline of a tool we wanted to use.
```
Comment (simurai) [inline _posts/2017-01-31-kap.md]: "Why did we build it?" should be a title.

Comment (zeke) [inline _posts/2017-01-31-kap.md]: It is actually a title already: " ## What is Kap and why did you create it?"

Kind of a two-parter. Maybe we should split it up into two questions?

**Comment by simurai** [comment]
Wow.. this is so awesome! ❤️ 

I always thought there should be more screenshots of apps, since Electron is "invisible", but screenshots might get out of date pretty quickly. Having them part of a blog post with a time stamp solves that problem. :v:

Also, the interview with the app's author is super interesting.

File: _posts/2017-01-31-kap.md (hunk 1 above)
Comment (simurai) [inline _posts/2017-01-31-kap.md]: Ohh.. hehe.. I see. They repeated the 2nd question in their answer, notice now the "we".

> Maybe we should split it up into two questions?

Yeah, might be more straightforward.

**Comment by zeke** [comment]
http://electron.atom.io/blog/2017/01/31/kap

https://twitter.com/electronjs/status/826489743017865221

🚀

**Pull request merged**